.github/workflows/sonarqube.yml

```
@@ -85,6 +85,8 @@ jobs:
           echo "REDIS_HOST=${{ secrets.REDIS_HOST }}" >> .env
           echo "REDIS_PORT=${{ secrets.REDIS_PORT }}" >> .env
           echo "REDIS_PASSWORD=${{ secrets.REDIS_PASSWORD }}" >> .env
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)